### PR TITLE
removing extra copybutton files

### PR DESCRIPTION
**Pull request by choldgraf**
- [x] Add / update documentation

I just realized that we had an old CSS and JS file from before the "sphinx-copybutton" plugin was a thing. This removes those files, and also updates the docs to use the same readthedocs config that binderhub uses (mostly). I'm gonna self merge this tomorrow AM if the tests pass since it may take some tweaking to get RTD to work properly :-/

### CODE REVIEW

File: docs/environment.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: tljh
+channels:
+  - conda-forge
+dependencies:
+- python=3.6
+- traitlets>=4.1
+- sphinx>=1.4, !=1.5.4
+- pip:
+  - recommonmark==0.4.0
```
Comment (yuvipanda) [inline docs/environment.yml]: we don't need recommonmark :)

**Comment by yuvipanda** [comment]
@choldgraf I'll leave this to you to self merge and iterate on!

File: docs/environment.yml (hunk 1 above)
Comment (choldgraf) [reply docs/environment.yml]: oops! ok removing that line and merging...I may merge a couple more to get RTD working if something is off

**Comment by yuvipanda** [comment]
@choldgraf I do! Did the entire theme change?!

**Comment by choldgraf** [comment]
yeah I dunno what's up with the theme changing, it happened with the other jupyter projects too. I wonder if this is an update to Alabaster?

**Comment by yuvipanda** [comment]
@choldgraf yeah, I *think* so. I love the new theme's typography! Except it's cutting the title of the project awkwardly :) Otherwise <3

**Comment by choldgraf** [comment]
haha yeah I saw that too, but I agree I think it is cleaner. We can always reduce the H1 size (or whatever that element is) by a little bit :-)

**Pull request merged**